### PR TITLE
Hi ahe mazi story

### PR DESCRIPTION
**Pull request by shashank2805**
Just for a practice

### CODE REVIEW

**Comment by shashank2805** [comment]
Please review this, @casey-collab

File: Marathi/mi_marathi/mi_marathi.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Mala khup anand zala marathi folder baghun.
```
Comment (casey-collab) [inline Marathi/mi_marathi/mi_marathi.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by shashank2805** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**